pkg/registrymirror/registrymirrorsstore.go

```diff
@@ -0,0 +1,325 @@
+package registrymirror
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/containers/image/v5/pkg/sysregistriesv2"
+	ciTypes "github.com/containers/image/v5/types"
+	"github.com/docker/distribution/reference"
+	configV1 "github.com/openshift/api/config/v1"
+	operatorV1Alpha1 "github.com/openshift/api/operator/v1alpha1"
+	"github.com/openshift/runtime-utils/pkg/registries"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/maputil"
+	"github.com/stackrox/rox/pkg/sync"
+	"github.com/stackrox/rox/pkg/utils"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	// default path to store registries config.
+	defaultRegistriesPath = "/var/cache/stackrox/mirrors/registries.conf"
```

Comment:
where did this come from? Is any of this configurable? As in, do we define something like `DefaultCache = `/var/cache`? If so, let's use those variables

Reply:
> where did this come from?

The volume is defined in the helm chart.

> Is any of this configurable?

Can certainly move the value into an environment variable, I didn't see the immediate benefit to doing so.

> As in, do we define something like DefaultCache = /var/cache`? If so, let's use those variables

No, the areas of Sensor that use `/var/cache/stackrox` all specify the path directly. ([ex1](https://github.com/stackrox/stackrox/blob/9418c9612db6d8c1a772173de93fa0cea466b685/sensor/common/sensor/helmconfig/cluster_id_cache.go#L12), [ex2](https://github.com/stackrox/stackrox/blob/ac551c488c8ed4dddb32e7ed2e5b5b6327629682/sensor/common/certdistribution/env.go#L7), [ex3](https://github.com/stackrox/stackrox/blob/ef5861d4da29c123ae0a4e008bbf7f7b454d4da9/scale/dev/launch_sensor.sh#L32))